Lib/_pyrepl/utils.py

```diff
@@ -10,6 +10,12 @@
 ZERO_WIDTH_TRANS = str.maketrans({"\x01": "", "\x02": ""})
 
 
+DEFAULT_PS1 = ">>> "
+DEFAULT_PS2 = ">>> "
```

Comment:
```suggestion
DEFAULT_PS2 = "... "
```

See https://docs.python.org/3/library/sys.html#sys.ps1
>sys.ps1
sys.ps2
Strings specifying the primary and secondary prompt of the interpreter. These are only defined if the interpreter is in interactive mode. Their initial values in this case are '>>> ' and '... '.

```
>>> import sys
>>> sys.ps1
'>>> '
>>> sys.ps2
'... '
>>> def foo():
...     pass
``` 

Maybe here the `DEFAULT_PS*` shall be used, too? Are there more places?
https://github.com/python/cpython/blob/246ed23456658f82854d518ad66d52652f992591/Lib/_pyrepl/simple_interact.py#L134-L135

IMHO, the comment in `reader.py` is wrong (or at least confusing):
https://github.com/python/cpython/blob/246ed23456658f82854d518ad66d52652f992591/Lib/_pyrepl/reader.py#L171-L178

Reply:
As @tomasr8 pointed above https://github.com/python/cpython/blob/95800fe6e719c829acf52fbb00198135b78719b4/Lib/_pyrepl/reader.py#L196-L199
there are different defaults too. So IDK what is really default :)

Reply:
> Maybe here the DEFAULT_PS* shall be used, too? Are there more places?

Yes, but I'm not sure that usage DEFAULT_PS* approved overall.

Reply:
IDK either, but IMHO we should **stick with what is documented and the status quo**, i.e. `DEFAULT_PS2 = "... "`.

The reason why 
https://github.com/python/cpython/blob/95800fe6e719c829acf52fbb00198135b78719b4/Lib/_pyrepl/reader.py#L196-L199
is not in effect:
https://github.com/python/cpython/blob/95800fe6e719c829acf52fbb00198135b78719b4/Lib/_pyrepl/main.py#L49-L54

and then later

https://github.com/python/cpython/blob/246ed23456658f82854d518ad66d52652f992591/Lib/_pyrepl/simple_interact.py#L134-L139

and then

https://github.com/python/cpython/blob/95800fe6e719c829acf52fbb00198135b78719b4/Lib/_pyrepl/readline.py#L375-L387

So IMHO we should consistently use `DEFAULT_PS1` and `DEFAULT_PS2` in those code parts above, but let @pablogsal decide on it.

Reply:
Moreover, this part
```python
reader.ps1 = ps1 
reader.ps2 = ps1 
reader.ps3 = ps2 
``` 
IMHO explains this confusion
https://github.com/python/cpython/blob/246ed23456658f82854d518ad66d52652f992591/Lib/_pyrepl/reader.py#L171-L178

ISTM, this is an implementation detail, and `ps2=ps1`, etc, maps to what we are used to in the cPython REPL.

Most probably, because the new PYREPL comes from PyPy?

Reply:
> Moreover, this part

Yeah, I saw this and this is a reason that I changed DEFAULT_PS2 yesterday.

> Most probably, because the new PYREPL comes from PyPy?

I'm not familiar with PyPy, so 🤷‍♂️